tests/src/burn.rs

```diff
@@ -544,6 +517,7 @@ fn should_not_burn_in_non_burn_mode() {
     support::assert_expected_error(error, 106, "InvalidBurnMode(106) must have been raised");
 }
 
+// This test is not really relevant anymore as approve_for_all does not check burnt tokens anymore
```

Comment:
```suggestion
// This test is no longer relevant as approve_for_all does not check burnt tokens anymore
```

Code comment, so probably doesn't need proofreading and you can safely ignore this review comment - just saw it while I was looking through.